x-pack/test/stack_functional_integration/apps/ccs/ccs_discover.js

```diff
@@ -203,5 +224,51 @@ export default ({ getService, getPageObjects }) => {
         expect(hitCount).to.be.lessThan(originalHitCount);
       });
     });
+
+    it('should add detection rule', async () => {
+      await supertest
+        .get('/api/detection_engine/rules/_find?filter=alert.attributes.name:CCS_Detection_test')
+        .expect(200)
+        .expect((res) => {
+          expect(res.body.data).eql([]);
+        });
+
+      await supertest
+        .post('/api/detection_engine/rules')
+        .set('kbn-xsrf', 'true')
+        .send({
+          description: 'This is the description of the rule',
+          risk_score: 17,
+          severity: 'low',
+          interval: '10s',
+          name: 'CCS_Detection_test',
+          type: 'query',
+          from: 'now-17520h',
+          index: ['data:.monitoring-es-*'],
+          timestamp_override: 'timestamp',
+          query: '*:*',
+          language: 'kuery',
+          enabled: true,
+        })
+        .expect(200);
+
+      await supertest
+        .get('/api/detection_engine/rules/_find?filter=alert.attributes.name:CCS_Detection_test')
+        .expect(200)
+        .expect((res) => {
+          expect(res.body.data).not.eql([]);
+        });
+    });
+
+    it('local:.siem-signals-(star) should discover data from the local cluster', async () => {
+      await PageObjects.common.navigateToApp('discover', { insertTimestamp: false });
+      await PageObjects.discover.selectIndexPattern('local:.siem-signals-*');
+      await PageObjects.timePicker.setCommonlyUsedTime('Today');
+      await retry.tryForTime(40000, async () => {
+        const hitCount = await PageObjects.discover.getHitCount();
+        log.debug('### hit count = ' + hitCount);
+        expect(hitCount).to.be.greaterThan('0');
```

Comment:
It is better if we assert the number of alerts we are expecting.

Reply:
This requires tuning, depends on the observation period and on the time elapsed when the reading is done. I also suspect that the observed index (.monitoring-es-*) get updated. Essentially, I'm not sure we can have a deterministic value here. Maybe I should investigate some other index for applying the rule onto.

Reply:
This is probably a good place to use esArchiver, but it's going to take a little bit of work.

The integration-test repo no longer uses node.js which would be required to call esArchiver.  
After the CCS config is provisioned we basically do `pushd ../kibana`, and then start the FTR tests.  

FTR tests do load data with esArchiver, but currently only to the local cluster.  While it's possible to add other configuration for FTR so that esArchiver could also load data in a remote cluster, I think it would be easier to use the CLI to call it from the jenkins_test.sh script in the integration-test just before starting the FTR.
If you want to go this route, let me know and I'll help with it.